spec/support/factory_girl.rb

```diff
@@ -9,4 +9,13 @@
     password_confirmation 'password'
   end
 
+  factory :game do
+    sequence(:name) { |n| "Stanton School Court ##{1}"}
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.<br>Space missing inside }.